### PR TITLE
feat(adirs): aircraft specific programmed adirs

### PR DESCRIPTION
**Pull request by Gurgel100**
Fixes #[issue_no]

## Summary of Changes

Adds aircraft specific programmed ADIRS.
The low speed warnings are programmable and are different for A320 and A380.

## Additional context

Discord username (if different from GitHub):

### A32NX
Check that the gear can be retracted and lowered below 260kts but not above 260kts.

### A380X
Check that in flight only one APU generator is available.
Check that only MFD and EWD are powered  on ground with batteries only. But in elec emergency config the captain's PFD and ND should also be powered.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

File: fbw-a32nx/src/wasm/systems/a320_systems/src/navigation.rs [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
+pub(crate) struct A320AirDataInertialReferenceSystemBuilder;
+impl A320AirDataInertialReferenceSystemBuilder {
+    pub(crate) fn build(context: &mut InitContext) -> AirDataInertialReferenceSystem {
+        let adirs_programming = AirDataInertialReferenceUnitProgramming::new(
+            Velocity::new::<knot>(350.),
+            MachNumber(0.82),
+            [
+                LowSpeedWarningThreshold::new(
+                    Velocity::new::<knot>(100.),
+                    Velocity::new::<knot>(4.),
```
[the remark this answers is not in the document]
Comment (Gurgel100) [reply fbw-a32nx/src/wasm/systems/a320_systems/src/navigation.rs]: Yep. Didn't have time yesterday. Done now.

**Review by tracernz — APPROVED** [review]

**Comment by tracernz** [comment]
Needs test instructions.

**Comment by alepouna** [comment]
^ Please add "QA Ready To Test" once instructions are added

**Comment by alepouna** [comment]
a bit too excited github actions arent you

**Comment by pilotseyea350** [comment]
Quality Assurance Tester Report

Discord Username  : PilotEyesA350 
Object of testing : #8983  
Aircraft          : A380X 
Tier of Testing   : 2
Date              : 29/09/2024 

Testing Process: 
1. Power BAT 1, ESS BAT, BAT 2 & APU BAT
2. Observe MFD and EWD powered up
3. Fly from EDDF to EDDF
4. When reaching 5000ft, Start APU
5. Turn all 4 generators off
6. Obersve APU GEN A in use
7. Turn APU OFF
8. Observe Captain PFD & ND powered up
9. Observer MFD and EWD powered up

Testing Results: 
Passed 

Negatives:       
N/A

Conclusions: 
1st A380W QA worked like a charm ! 😃 

Media:
N/A

**Comment by tylerclt** [comment]
Quality Assurance Trainee Report

Discord Username: tylerb780 
Object of testing: #8983 
Aircraft: A32NX 
Tier of Testing: 1
Date: 07/10/2024 

Testing Process: 
1. Loaded into KASE on the runway with engines running
2. Departed
3. Climbed and leveled off at 11000ft 
4. Lowered and retracted the gear at 240kts (Below 260kts)
5. Accelerated to 270kts and verified the gear could not extend
6. Decelerated below 260kts and extended the gear then accelerated to 270kts
7. Verified that the gear did not retract

Testing Results: 
Passed 

Conclusions: 
Works as intended!

**Comment by cmoore993** [comment]
Quality Assurance Trainee Report

Discord Username: gamer_guy954
Object of testing: https://github.com/flybywiresim/aircraft/pull/8983
Aircraft: A380-842
Tier of Testing: 1
Date: 07/10/2024

Process: 
Load onto KPHL Ramp 100

Result: aircraft was invisible both in the cockpit and in the external views.

**Comment by Popespice** [comment]
> Quality Assurance Trainee Report
> 
> Discord Username: gamer_guy954 Object of testing: #8983 Aircraft: A380-842 Tier of Testing: 1 Date: 07/10/2024
> 
> Process: Load onto KPHL Ramp 100
> 
> Result: aircraft was invisible both in the cockpit and in the external views.

Working as intended, everything in the plane will be invisible without the model.

**Pull request merged**